recipe/meta.yaml

```diff
@@ -38,9 +38,9 @@ test:
     - pytest
   imports:
     - numcodecs
-    - numcodecs.blosc
-    - numcodecs.lz4
-    - numcodecs.zstd
+    - numcodecs.blosc  # [python_impl != "pypy"]
+    - numcodecs.lz4    # [python_impl != "pypy"]
+    - numcodecs.zstd   # [python_impl != "pypy"]
```

Comment:
Skipping these `import` tests on PyPy as we [don't build them on PyPy]( https://github.com/zarr-developers/numcodecs/blob/b7c5a81407422b00a16b6eba3c63def335be51d5/setup.py#L348-L349 ). More details in issue ( https://github.com/zarr-developers/numcodecs/issues/238 ).

Reply:
@isuruf, could you please double check and let me know if I'm missing something here. It seems that [CI is still running these `import` tests on PyPy]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=207660&view=logs&jobId=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&j=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&t=841356e0-85bb-57d8-dbbc-852e683d1642 ).

Reply:
See the docs at https://conda-forge.org/docs/maintainer/knowledge_base.html#pypy-builds

@conda-forge-admin, rerender

Reply:
Yep saw that. Though missed the re-rendering bit. Thanks! 😄